ui/package.json

```diff
@@ -14,8 +14,8 @@
   },
   "scripts": {
     "dev": "vite --config vite.config.dev.ts",
-    "build": "vue-tsc --noEmit && vite build",
-    "preview": "vite preview --port 5002"
+    "build": "vue-tsc --noEmit && vite build && cp dist/* ../features/ui/src/main/resources/ui-ext/ ",
```

Comment:
Just FYI, this won't work for most windows terminals, but I guess we can force them to build in wsl or git bash